packages/opentelemetry-tracing/src/Span.ts

```diff
@@ -60,14 +66,20 @@ export class Span implements api.Span, ReadableSpan {
     kind: api.SpanKind,
     parentSpanId?: string,
     links: api.Link[] = [],
-    startTime: api.TimeInput = hrTime()
+    startTime?: api.TimeInput
   ) {
     this.name = spanName;
     this.spanContext = spanContext;
     this.parentSpanId = parentSpanId;
     this.kind = kind;
     this.links = links;
-    this.startTime = timeInputToHrTime(startTime);
+
+    if (startTime != null) {
```

Comment:
i believe we should check `startTime` for `undefined` instead of `null`

Reply:
Mind the missing second `=` - `startTime !=  null` is the same as `(startTime !== null) || (startTime !== undefined)`.

Reply:
Didn't see that indeed, i would prefer to have strict check so its easier to read the code though